mobile/ios/ios-emulator-services.ts

```diff
@@ -220,5 +217,24 @@ class IosEmulatorServices implements Mobile.IiOSSimulatorService {
 		let runningSimulatorId = runningSimulator.id;
 		return runningSimulatorId;
 	}
+
+	private killApplication(applicationName: string): IFuture<void> {
+		return (() => {
+			try {
+				this.$childProcess.exec(`killall ${applicationName.split(".")[0]}`).wait();
+			} catch(e) {
+				this.$logger.trace(`Unable to kill simulator: ${e}`);
```

Comment:
When this code cannot kill the simulator, the execution continues and we will try to start the new process. Is this the intended behavior?